packages/application-shell/src/configure-store.ts

```diff
@@ -18,16 +29,37 @@ import {
   selectUserId,
 } from './utils';
 
-const mergeObjectValues = object =>
-  Object.entries(object).reduce((acc, [, value]) => ({ ...acc, ...value }), {});
+interface ApplicationWindowWithDevtools extends ApplicationWindow {
+  __REDUX_DEVTOOLS_EXTENSION__: (config?: {
+    actionsBlacklist: string[];
+  }) => StoreEnhancer<unknown, {}>;
+}
+declare let window: ApplicationWindowWithDevtools;
+
+type TEnhancedStore = {
+  injectedReducers: { [key: string]: ReducersMapObject };
+  injectReducers: (payload: {
+    id: string;
+    reducers: ReducersMapObject;
+  }) => void;
+  removeReducers: (payload: { id: string }) => void;
+};
+
+const identityStoreEnhancer: StoreEnhancer<unknown, {}> = noop => noop;
+
+const mergeObjectValues = <T>(object: { [key: string]: T }) =>
+  Object.entries<T>(object).reduce(
+    (acc, [, value]) => ({ ...acc, ...value }),
+    {}
+  );
 
 const patchedGetCorrelationId = () =>
   getCorrelationId({
     userId: selectUserId({ apolloCache: apolloClient }),
   });
 
-const createInternalReducer = (injectedReducers = {}) =>
-  combineReducers({
+const createInternalReducer = (injectedReducers: ReducersMapObject = {}) =>
+  combineReducers<unknown>({
```

Comment:
This is a bit of a "workaround" because of this:

<img width="690" alt="image" src="https://user-images.githubusercontent.com/1110551/66388950-832c6800-e9c7-11e9-9c43-09e7910e1fa8.png">

Reply:
I think I know I'm familiar with this problem a bit too well.
code comment that this is a workaround and what the original problem was?

Reply:
Yeah I can do that for now, maybe we can find a better solution in the future.